app/controllers/taxon_changes_controller.rb

```diff
@@ -357,6 +357,37 @@ def group
       format.html { render layout: "bootstrap" }
     end
   end
+
+  def analysis_with_url( row )
+    if row[:id_count] == 0
+      url = nil
+    else
+      observation_params = {
+        id: row[:id_obs].join(",")
+      }
+      url = observations_path( observation_params )
+    end
+    return { taxon_id: row[:taxon_id], id_count: row[:id_count], url: url }
+  end
+
+  def analyze_ids
```

Comment:
Since this is kind of a performance issue, it should require auth, and maybe require a curator.